src/components/DisplayPhotos.js

```diff
@@ -0,0 +1,30 @@
+const DisplayPhotos = (props) => {
```

Comment:
seems like you're using components and props a different way than I did which is really cool to see! I wonder how it would have worked if you created a two seperate components for your image and quote generation.